buildSrc/src/main/java/FileExt.kt

```diff
@@ -0,0 +1,12 @@
+import java.io.File
+import java.util.*
+
+fun File.loadGradleProperties(fileName: String): Properties {
```

Comment:
should we create a directory `extensions` and put this file into it?

Reply:
@manh-t I think we don't need to move this into a new extensions package. 
- I don't see many people do that
  - https://github.com/gradle/kotlin-dsl-samples/blob/master/samples/multi-kotlin-project-with-buildSrc/buildSrc/src/main/kotlin/utils.kt
  - https://github.com/gradle/kotlin-dsl-samples/tree/master/samples/project-with-buildSrc/buildSrc/src/main/kotlin
- IMO, this is a simple buildSrc module so keep all classes at root package still acceptable. Moving it into a new package produces more required `import` added.

![image](https://user-images.githubusercontent.com/16315358/204756705-ac34f04b-ce8c-4609-a67e-8f0d38adc87c.png)

As this IC focuses more on KMM, please allow me ignore this issue 🙏

Reply:
@luongvo okay then